### DISCUSSION TITLE
Registration of streamlines with SRI atlas

### DISCUSSION THREAD

**Discussion by Sunita2023** [dipy/dipy#3160] [Tractography / Tractometry]
Dear Serge,

We could register streamlines with the skull stripped T1W image which was registered to MNI. There was no problem. 
However, when we registered with SRI atlas the generated streamlines are misshaped. Looks like as if it flattened. Basically we want streamlines to fit into SRI shape. 

Could you please suggest how we can do this.  

Thank you so much for your help. 

Best regards,
Sunita

**Comment by Sunita2023** [comment]
Hi Serge,

You are expert in the field of DIPY. Your guidance would be of great help. 
 Is it possible to generate streamlines which fit into the SRI atlas?

Awaiting your reply. 
Thanks a lot.

Best regards,
Sunita

**Comment by skoudoro** [comment]
Hi @Sunita2023,

What is SRI atlas and SRI shape ? Sorry, I do not know this atlas.

>  Looks like as if it flattened

flattened problem are usually due to a problem with voxel size. something that you could check and a resample (`reslice` function in DIPY) might be needed

**Comment by Sunita2023** [comment]
Hi Serge,

SRI24 is a brain atlas. You can find the reference article in the following link.
https://pubmed.ncbi.nlm.nih.gov/20017133/
shape of SRI24 is (240 240 155).

We have T1W 3D image and its tumor mask of shape 240 x 240 x 155. Now we want to register both image and mask to MNI152 template. MNI152 (mni_icbm152_t1_nlin_asym_09a) shape is 197 x 233 x 189. We are doing linear registration and image is registered but it seems mask is not registered. How do to registration and resampling of the tumor mask so that it will match with the registered image?

Thank you so much.

Best regards,
Sunita
